src/hypergraph.jl

```diff
@@ -295,4 +295,33 @@ function nhv(h::Hypergraph{T, V, E}) where {T <: Real, V, E}
     length(h.v2he)
 end
 
+"""
+    nhv(h::Hypergraph{T, V, E}) where {T <: Real, V, E}
```

Comment:
wrong function name `nhv`, no need to pass the whole signature. The docstring should be exactly above the function to which it refers.